test/WebMotions.Fake.Authentication.JwtBearer.Tests/FakeJwtBearerTests.cs

```diff
@@ -248,6 +249,18 @@ public async Task SetClaimIssuerOverride()
             response.ResponseText.Should().Be("SuperUserName");
         }
 
+        [Fact]
+        public async Task CanSendClaimsViaJwt()
```

Comment:
I think you could just add another parameter to the CreateServer method with `string claimType = ClaimTypes.NameIdentifier` and use that parameter on line 297

It would remove the extra CreateServer you created. You can also configure the options when creating the server so you would be able to pass the `options.BearerValueType = FakeJwtBearerBearerValueType.Jwt`

It will keep the code tight and avoid duplication

Reply:
You dont need to add an extra path to the server.
All you need is:
```csharp
private static TestServer CreateServer(Action<FakeJwtBearerOptions> options = null, Func<HttpContext, Func<Task>, Task> handlerBeforeAuth = null, string claimType = ClaimTypes.NameIdentifier)
```

then
```csharp
var identifier = context.User.FindFirst(ClaimTypes.NameIdentifier);
```

becomes
```csharp
var identifier = context.User.FindFirst(claimType);
```
and in your test you do:
```csharp
var response = await SendAsync(client, "http://example.com/oauth");
```